sensus/sensus/code.py

```diff
@@ -1,9 +1,3 @@
-# We need three layers. Input neurons are the intensities of the pixels: i x j pixels means i x j input neurons.
-# 1) 28 x 28 = 784 dimensional input vector, which is the image of a single digit. Each value in this vector 
-# indicates what shade of grey a pixel is between white and black. y is a 10 dimensional output vector, one value 
-# for each digit between 0-9 if the network decides on 5 for a given training image, the output vector will be 
-# y(x) = (0, 0, 0, 0, 1, 0, 0, 0, 0, 0) transposed. 
-
 import numpy as np
```

Comment:
i was just a bit confused wit this file. It might be because im not used to github but why are there these 2 files with the same content?